moduleframework/mtf.yaml

```diff
@@ -0,0 +1,103 @@
+# default module type in case you don't wath everytime use MODULE=docker or define it in config.yaml
+default_module: docker
+
+# openshift specific section
+openshift:
+  init_wait: 50
+  docker_registry: "docker-registry"
+  project: 'project'
+  template: 'template'
+  ip:
+  user:
+  password:
+  local_ip: "127.0.0.1"
+  local_user: "developer"
+  local_password: "developer"
+
+# docker specific section
+docker:
+# default name of Docker file
+  dockerfile: "Dockerfile"
+# default name of HeplMD file
+  helpmdfile: "help.md"
+# default location of these files (based on axiom that you are in tests/ subdir)
+  dockerfiledefaultlocation: "../"
+
+# modularity specific  section
+modularity:
+  tempmodulefile: "tempmodule.yaml"
+  default_module_stream: "master"
+  default_profile: "default"
+
+# systemd-nspawn config
+nspawn:
+# default location where images (directories) lives
+  basedir: "/opt"
+
+# generic section mainly contains timeouts
+generic:
+# default architecture
+  arch: "x86_64"
+# command to find which packager to use (yum, dnf, apt-get)
+  packager_cmd: "test -e /usr/bin/dnf && echo 'dnf -y' ||
+   ( test -e /usr/bin/microdnf && echo 'microdnf'      ||
+   ( test -e /usr/bin/yum && echo 'yum -y'             ||
+   echo 'apt-get -y' ) )"
+# which values are considered as true
+  true_values: ['yes', 'YES', 'yes', 'True', 'true', 'ok', 'OK']
+# name of directory with static tests - linters
+  static_tests: 'static'
+# name of directory with generic tests - like rpm signarure check
+  generic_tests: 'generic'
+  # default timeout in secs of processes
+  processtimeout: 120
+# number of retry for @retry decorator
+  retrycount: 3
+  # default timeout in secs of for retry decorator
+  retrytimeout: 30
+  # default timeout to start nspawn container
+  nspawn_timeout: 10
+
+# pdc section, location of pdc server
+pdc:
+  pdc_server: "https://pdc.fedoraproject.org/rest_api/v1/modules"
+  # internal
+  #pdc_server: "https://pdc.engineering.redhat.com/rest_api/v1/modules"
+
+compose:
+  repomd: "repodata/repomd.xml"
+# basic repository for nspawn, RELEASE and ARCH will be replaced by .format function
+  baseurlrepo: "https://download.fedoraproject.org/pub/fedora/linux/releases/{RELEASE}/Workstation/{ARCH}/os"
+
+# openidc section used by ODCS for autentication against fedora
+openidc:
+  token:
+  auth: ["odcs", "https://id.fedoraproject.org/openidc/", {Token: "Token", Authorization: "Authorization"},"odcs-authorizer","notsecret"]
+  scopes: ['openid', 'https://id.fedoraproject.org/scope/groups', 'https://pagure.io/odcs/new-compose', 'https://pagure.io/odcs/renew-compose', 'https://pagure.io/odcs/delete-compose']
+
+# ODCS section
+odcs:
+  compose_type: "module"
+  # timeout in secs, how long wait for compose (composes are cached, but first round can take long time (createrepo of packages))
+  timeout: 1200
+  new_compose_dict: { sigkeys: [''] }
+  url: "https://odcs.fedoraproject.org"
```

Comment:
I expect that internally we'll have a different config.

Reply:
Yes, it is also mentioned few lines under (location of internal odcs server)
Probably we will ship, or somewhere stored config file for downstream testing